src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(DifferentColors.values().length);
+        DifferentColors ballColor = DifferentColors.values()[index];
+        return ballColor.toString();
```

Comment:
Instead of using `toString()`, it's better to use `name()` for getting the String representation of enum constants to avoid issues if `toString()` is overridden.